phivk/gradu.al
# Allows build tool to handle multiple sheets.

This gets the sheets from a current spreadsheet and iterate over those.

This will work for single or multiple sheets and is backward compatible.

To test:
- Run the build command
- Confirm it runs without error
- Confirm the viz works

- In `communities.js` update to reflect the demo sheet:
```
module.exports = [
  {
    name: "demo",
    spreadsheetId: "1QXhm6lxtSciNsREDz6Y-Qf1bmB9n5bm2HUvaf19j9lg",
  },
  // {
  //   name: "mozfest",
  //   spreadsheetId: "1Y1QE7pZN97QtuDCDfpGqP5c1gMeKmPKGFnSUgtEQZbk",
  // },
];
```
- Run the build script
- Confirm that the nodes.json has info from both sheets of the spreadsheet (https://docs.google.com/spreadsheets/d/1QXhm6lxtSciNsREDz6Y-Qf1bmB9n5bm2HUvaf19j9lg/edit#gid=1493340608)
- Confirm there are no errors
- Confirm that the data viz looks as expected (with data from both sources)
